src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayRate = 40 * days;
+  const twentyOff = 20;
+  const fiftyOff = 50;
+
+  switch (days) {
```

Comment:
According to checklist item 3, you should use `if` instead of `while`.

Reply:
[CODE STYLE] - You should return the value immediately in if statements. And don't use else after return statement. If return is performed - code after it won't be executed anyway.    

I'm sorry but I don't see that part. What do you mean?